Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py

```diff
@@ -0,0 +1,253 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# IMPORTS
+import requests
+from typing import List
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+SOURCE_NAME = "AutoFocusFeedDaily"
+DAILY_FEED_BASE_URL = 'https://autofocus.paloaltonetworks.com/api/v1.0/output/threatFeedResult'
```

Comment:
if this var is used only once, no need for global definition.
you can define it in the main with the other params and pass it in the client